[2025-09-20T12:06:53.422+0200] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: manual_full_load.load_fct_sales manual__2025-09-20T10:06:48.685593+00:00 [queued]>
[2025-09-20T12:06:53.431+0200] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: manual_full_load.load_fct_sales manual__2025-09-20T10:06:48.685593+00:00 [queued]>
[2025-09-20T12:06:53.432+0200] {taskinstance.py:1308} INFO - Starting attempt 1 of 2
[2025-09-20T12:06:53.444+0200] {taskinstance.py:1327} INFO - Executing <Task(PostgresOperator): load_fct_sales> on 2025-09-20 10:06:48.685593+00:00
[2025-09-20T12:06:53.448+0200] {standard_task_runner.py:57} INFO - Started process 7624 to run task
[2025-09-20T12:06:53.451+0200] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'manual_full_load', 'load_fct_sales', 'manual__2025-09-20T10:06:48.685593+00:00', '--job-id', '125', '--raw', '--subdir', 'DAGS_FOLDER/manual_full_load.py', '--cfg-path', '/tmp/tmp2tjfkr63']
[2025-09-20T12:06:53.453+0200] {standard_task_runner.py:85} INFO - Job 125: Subtask load_fct_sales
[2025-09-20T12:06:53.501+0200] {task_command.py:410} INFO - Running <TaskInstance: manual_full_load.load_fct_sales manual__2025-09-20T10:06:48.685593+00:00 [running]> on host 1102361e702e
[2025-09-20T12:06:53.607+0200] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='manual_full_load' AIRFLOW_CTX_TASK_ID='load_fct_sales' AIRFLOW_CTX_EXECUTION_DATE='2025-09-20T10:06:48.685593+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-09-20T10:06:48.685593+00:00'
[2025-09-20T12:06:53.608+0200] {sql.py:265} INFO - Executing: 
        INSERT INTO dwh.fct_sales (date_id, store_id, product_id, qty, revenue)
        SELECT sale_date, store_id, product_id, qty, revenue
        FROM staging.sales_stg;
        
[2025-09-20T12:06:53.629+0200] {base.py:73} INFO - Using connection ID 'postgres_retail' for task execution.
[2025-09-20T12:06:53.699+0200] {base.py:73} INFO - Using connection ID 'postgres_retail' for task execution.
[2025-09-20T12:06:53.707+0200] {sql.py:375} INFO - Running statement: 
        INSERT INTO dwh.fct_sales (date_id, store_id, product_id, qty, revenue)
        SELECT sale_date, store_id, product_id, qty, revenue
        FROM staging.sales_stg;
        , parameters: None
[2025-09-20T12:06:53.731+0200] {sql.py:384} INFO - Rows affected: 250
[2025-09-20T12:06:53.746+0200] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=manual_full_load, task_id=load_fct_sales, execution_date=20250920T100648, start_date=20250920T100653, end_date=20250920T100653
[2025-09-20T12:06:53.793+0200] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2025-09-20T12:06:53.826+0200] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
